DittoToolsApp/DittoToolsApp.xcodeproj/project.pbxproj

```diff
@@ -620,7 +627,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = dittolive.DittoToolsApp;
+				PRODUCT_BUNDLE_IDENTIFIER = dittolive.DittoToolsApp.WE;
```

Comment:
I think this is a test ID you were using and can be reverted? (here and above)

Reply:
Good catch